### PR TITLE
Adding ADK Adapter Style console example in Java.

### PR DESCRIPTION
**Pull request by pujanm**

### CODE REVIEW

File: adk-examples/Java/falkonryadk/.idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (phagunbaya) [inline adk-examples/Java/falkonryadk/.idea/misc.xml]: remove `.idea` folder

File: adk-examples/Java/falkonryadk/pom.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.pujan</groupId>
```
Comment (phagunbaya) [inline adk-examples/Java/falkonryadk/pom.xml]: `com.example`

File: adk-examples/Java/falkonryadk/src/main/java/ADKConnector.java [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+    Field field = new Field();
+    TimeObject time = new TimeObject();
+    Signal signal = new Signal();
+    String datastreamId;
+
+    ADKConn() throws Exception {
+    }
+
+    static Logger log = Logger.getLogger(ADKConn.class);
+    public String createDataStream() throws Exception {
```
Comment (phagunbaya) [inline adk-examples/Java/falkonryadk/src/main/java/ADKConnector.java]: Java doc missing http://drjava.org/docs/user/ch10.html

File: adk-examples/Java/falkonryadk/target/classes/log4j.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (phagunbaya) [inline adk-examples/Java/falkonryadk/target/classes/log4j.xml]: add `target` to gitignore

File: adk-examples/Java/falkonryadk/src/main/java/FileAdapter/FileAdapter.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        if(f.isFile()){
+            String fileType = FilenameUtils.getExtension(fileName);
+            if((fileType.toLowerCase().equals("csv"))||(fileType.toLowerCase().equals("json"))){
+                try {
+                    FileReader fr = new FileReader(f);
+                    BufferedReader br = new BufferedReader(fr);
+                    String s = IOUtils.toString(br);
+                    return s;
+                } catch (IOException e) {
+                    e.printStackTrace();
```
Comment (phagunbaya) [inline adk-examples/Java/falkonryadk/src/main/java/FileAdapter/FileAdapter.java]: use log4j

File: adk-examples/Java/falkonryadk/src/main/java/ADKConnector.java [hunk 6]
```diff
@@ -0,0 +1,342 @@
...
+
+        falkonry.addInputStream(datastreamId, stream, options);
+    }
+
+    public void getLiveOutput(String assessmentId) throws Exception {
+        BufferedReader outputBuffer;
+        outputBuffer = falkonry.getOutput(assessmentId);
+        int i;
+        while((i=outputBuffer.read())!=-1){
+            System.out.print((char)i);
```
Comment (phagunbaya) [inline adk-examples/Java/falkonryadk/src/main/java/ADKConnector.java]: log4j

File: adk-examples/Java/falkonryadk/src/main/java/ADKConnectorHist.java [hunk 7]
```diff
@@ -0,0 +1,534 @@
...
+
+    #################### For creating datastream and adding historical data ################
+
+    The below code will create a datastream and post the historical data as a string to the
+    datastream. You will have to give the data as a string or give the fileName and pass it
+    to the getData() method of the file adapter.
+
+*/
+
+//        final ADKConnHist adk = new ADKConnHist();
```
Comment (phagunbaya) [inline adk-examples/Java/falkonryadk/src/main/java/ADKConnectorHist.java]: Is there a need to comment all of this code?

File: adk-examples/Java/falkonryadk/src/main/java/ADKConnectorLive.java [hunk 8]
```diff
@@ -0,0 +1,288 @@
...
+    NOTE:-
+     1. Go on the Falkonry UI and build a model.
+     2. After building a model click LIVE(OFF) button to turn on the LIVE INPUT.
+     While using this method you can choose between ingestData(), ingestDataFromFile() and ingestDataFromFolder()
+     and set live parameter as true.
+     3. CHANGE THIS AS PER YOUR WORKFLOW
+
+*/
+
+//    final ADKConn adk = new ADKConn();
```
Comment (phagunbaya) [inline adk-examples/Java/falkonryadk/src/main/java/ADKConnectorLive.java]: Should not be commented.

**Comment by phagunbaya** [comment]
There should be a README explaining the structure and some brief about the available methods.

**Comment by phagunbaya** [comment]
Any PR should not be merged by PMs except for adk-documentation repo. Also, code review changes were pending.

**Comment by jeegs6up** [comment]
Ok. Sorry. I had reviewed this with the team and was under the impression I was driving this closure. The actions suggested prior were addressed and I merged these PRs which were pending for a few days. I took the adk examples as an extension of the documentation. Will keep in mind next time.

**Pull request merged**